sirepo/package_data/static/json/activait-schema.json

```diff
@@ -422,6 +431,7 @@
             "dimensionality": ["Dimensionality", "Integer", 32],
             "layer": ["Layer", "NeuralNetLayer", "Conv2D"],
             "kernel" : ["Kernel Size", "Integer", 3],
+            "kernel_initializer": ["Kernel Initializer", "KernelInitializer", "GlorotUniform"],
```

Comment:
There needs to be a fixup in sim_data.activait to add this recursively to the layers. Running through the neuralNet.layers field and into the concatenate child models, calling update_model_defaults() on each.

To reproduce the problem, go to the master branch, discard changes to the Pet db example and open a new copy. Then switch to the new branch, and open the same example.